woocommerce/woocommerce-ios
# Update release-toolkit to fix `pot` generation

## Description

This will save us from having to do the metadata re-generation dance from https://github.com/woocommerce/woocommerce-ios/pull/10545 (15.1) in this new release (15.2).

## Testing instructions
If you run `bundle exec fastlane update_appstore_strings` on `release/15.2`, you'll see it modifies the `pot` file removing trailing `\n` in multi-line strings:

<img width="1728" alt="image" src="https://github.com/woocommerce/woocommerce-ios/assets/1218433/8c6a6878-ef19-4dbc-8aa6-49c7909cb4d9">

Running the lane on this branch does not edit the file.

## Screenshots
N.A.


---
